src/org/labkey/test/tests/component/GridPanelTest.java

```diff
@@ -1709,14 +1716,14 @@ public void testFilterDialogWithViews() throws IOException, CommandException
         log("Go back to default view. Filter on a column to be removed, then change view that does not include the column.");
         grid.selectView(VIEW_DEFAULT);
 
-        log(String.format("Filter column '%s' to true.", FILTER_BOOL_COL));
-        grid.filterColumn(FILTER_BOOL_COL, Filter.Operator.EQUAL, true);
+        log(String.format("Filter column '%s' to true.", FILTER_BOOL_COL.getName()));
+        grid.filterColumn(FILTER_BOOL_COL.getName(), Filter.Operator.EQUAL, true);
```

Comment:
I updated `FieldInfo` recently. It can now be used directly for this sort of thing.
```suggestion
        grid.filterColumn(FILTER_BOOL_COL, Filter.Operator.EQUAL, true);
```

Reply:
Great. Done.